experimental/tiledb/common/dag/data_block/test/unit_blocks_and_ports.cc

```diff
@@ -31,6 +31,7 @@
  *
  */
 
+#ifndef _MSC_VER
```

Comment:
How bad is this MSVC issue? Using this big of a hammer on tests seems a bit concerning to me. I'm worried we'll code ourselves into a corner and end up with a feature that can't be used on MSVC which AFAIK is one of our supported compilers.

Reply:
Well, the ones I commented out are those kinds of errors that spew pages of error messages.  I fought with one of them for half a day ("sieve").  I think the "sieve" tests/examples in general are integration rather than unit tests (which might be a reason to keep them).  But they are a) crufty because they were b) proofs of concepts for c) integration and d) performance.  I can try a bit longer (just a bit) to make these work, but I find the msvc error messages not to be very helpful -- and the errors don't show up in g++ or clang. 

There is another story to fix these.

Reply:
More than fair. Not bothering to fix things that might not exist in the future is good enough for me.

Reply:
I fixed the pool allocator and data block problems in a different branch (redid their implementation) and then went back and patched this branch with some small fixes for those.  The `Edge` issues in `unit_task_graph` and the whatever they are issues in `nodes_sieve` I am not so sure about.